src/Cli/dotnet/commands/dotnet-workload/WorkloadCommandParser.cs

```diff
@@ -29,9 +29,16 @@ public static Command GetCommand()
             return Command;
         }
 
-        internal static void ShowWorkloadsInfo(ParseResult parseResult, IWorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null)
+        internal static void ShowWorkloadsInfo(ParseResult parseResult = null, IWorkloadInfoHelper workloadInfoHelper = null, IReporter reporter = null)
         {
-            workloadInfoHelper ??= new WorkloadInfoHelper(parseResult.HasOption(SharedOptions.InteractiveOption));
+            if(workloadInfoHelper != null)
+            {
+                workloadInfoHelper ??= new WorkloadInfoHelper(parseResult.HasOption(SharedOptions.InteractiveOption));
```

Comment:
`parseResult` here could be null, might need to guard this access with a check.

Reply:
Does the if condition below this line work as a check?